docs/installation.md

```diff
@@ -63,6 +64,26 @@ helm upgrade my-datadog-operator datadog/datadog-operator -f values.yaml
    helm upgrade my-datadog-operator datadog/datadog-operator -f values.yaml
    ```
 
+### Configure cluster name (optional)
+
+Setting a cluster name is optional but recommended. The cluster name can be configured in the following ways:
+
+1. **Helm chart `clusterName` value** (sets `DD_CLUSTER_NAME` environment variable on the Operator):
```

Comment:
```suggestion
-  **Helm chart `clusterName` value** (sets `DD_CLUSTER_NAME` environment variable on the Operator):
```

For steps that don't have a hierarchy (and can be used interchangeably) use an unordered list!